pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -177,14 +177,27 @@ func (k *Bootstrapper) init(cfg config.ClusterConfig) error {
 		"FileAvailable--etc-kubernetes-manifests-etcd.yaml",
 		"Port-10250", // For "none" users who already have a kubelet online
 		"Swap",       // For "none" users who have swap configured
-		"SystemVerification",
 	}
 	ignore = append(ignore, bsutil.SkipAdditionalPreflights[r.Name()]...)
 
+	skipSystemVerification := false
 	// Allow older kubeadm versions to function with newer Docker releases.
+	if version.LT(semver.MustParse("1.13.0")) {
+		glog.Info("ignoring SystemVerification for kubeadm because of kubernetes version")
+		skipSystemVerification = true
+	}
+	if driver.BareMetal(cfg.Driver) && r.Name() == "Docker" {
+		if v, err := r.Version(); err == nil && strings.Contains(v, "azure") {
+			glog.Info("ignoring SystemVerification for kubeadm because of docker version")
```

Comment:
change the log, to mention azure ! 
'because using azure version of docker"

Reply:
OK, we didn't mention the old k8s version either